src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = new Ball[3];
+        int ballsLength = balls.length;
+
+        for (int i = 0; i < ballsLength; i++) {
+            balls[i] = new Lottery().getRandomBall();
+            System.out.println(balls[i].toString());
```

Comment:
```suggestion
            System.out.println(balls[i]);
```
under the hood it will be called automatically